syntax = "proto3";

message StopStream {

}
